Доработать калькулятор из методички

VladimirVV1987 opened the pull request:
Урок 3. Операторы и управляющие конструкции, базовые функции
1. Доработать калькулятор из методички: больше операций и валидации данных 
(проверка деления на 0, возведение в степень, факториал) + форматирование строки при выводе дробного 
числа ( 2 знака после точки)

VladimirVV1987 committed:
1. Доработать калькулятор из методички

v-lozhkin commented on hw3/main.go:
```diff
@@ -0,0 +1,72 @@
+		if b == 0 {
+			fmt.Println("Делить на ноль нельзя!")
+		}
+		res = a / b
```
На ноль деление проверили, но все равно делите :)

VladimirVV1987 committed:
4,1. Познакомьтесь с алгоритмом сортировки вставками. Напишите приложение, которое принимает на вход набор целых чисел и
отдаёт на выходе его же в отсортированном виде.

VladimirVV1987 committed:
первая домашка

VladimirVV1987 committed:
Урок 4. Сложные типы данных: массивы, слайсы и мапы
1. Познакомьтесь с алгоритмом сортировки вставками. Напишите приложение, которое принимает на вход набор целых чисел и
отдаёт на выходе его же в отсортированном виде.
Сохраните код, он понадобится нам в дальнейших уроках.

VladimirVV1987 committed:
111

VladimirVV1987 committed:
rm

VladimirVV1987 committed:
test

VladimirVV1987 committed:
test

VladimirVV1987 committed:
Урок 5.

VladimirVV1987 committed:
Урок 8

v-lozhkin commented on hw8/main.go:
```diff
@@ -0,0 +1,35 @@
+
+package main
+import (
+"flag"
```
не отформатировано

v-lozhkin replied:
И эти импорты не используются

v-lozhkin commented on hw8/config/config.go:
```diff
@@ -0,0 +1,68 @@
+}
+
+func (c *SrvConfigs) DefaultConfigs() {
+	matchAppPort, _ := regexp.MatchString(`[0-9]*`, c.AppPort)
```
Вообще говоря, регулярку можно один раз скомпилировать, а потом просто переиспользовать

v-lozhkin commented on hw8/config/go.mod:
```diff
@@ -0,0 +1,5 @@
+module config
```
Почему это отдельный модуль?

v-lozhkin commented on hw8/main.go:
```diff
@@ -0,0 +1,35 @@
+func main() {
+	var c config.SrvConfigs
+	c.AddSrvConfigs()
+	config.SrvConfigs(&c)
```
что это за конструкция?

v-lozhkin commented on hw8/config/config.go:
```diff
@@ -0,0 +1,68 @@
+		SentryUrl: 			os.Getenv("Sentry_Url"),
+		KafkaBroker:       os.Getenv("KAFKA_BROKER"),
+		SomeAppID:         os.Getenv("SOME_APP_ID"),
+		SomeAppkEY:         os.Getenv("SOME_APP_KEY"),
```
у структуры нет такого поля

VladimirVV1987 committed:
Домашка 9
